src/Files.Uwp/Views/Pages/PropertiesGeneral.xaml

```diff
@@ -199,6 +199,36 @@
                             Visibility="{x:Bind ViewModel.ItemSizeProgressVisibility, Mode=OneWay}" />
                     </Grid>
 
+                    <!--  Uncompressed Item size  -->
+                    <TextBlock
+                        Grid.Row="2"
```

Comment:
Can you increase the row count for all the items following this? Otherwise, this overlaps with the next row.